apps/system/js/app_chrome.js

```diff
@@ -791,6 +791,12 @@
     };
 
   AppChrome.prototype.handleScrollAreaChanged = function(evt) {
+    // Make sure the scroll-area-changed is coming from the right element.
+    if (evt && (!this.app || !this.app.browser ||
```

Comment:
That's what I thought, but I saw a null this.app while testing this with web activities, so... I dunno what's going on there but I'd rather keep it in :)